### ISSUE TITLE
Geo Improve Testing Project Name Issue

### ISSUE THREAD

**Issue by FLO-2DKaren** [FLO-2DSoftware/qgis-flo-2d-plugin#1040]
When you open QGIS the name of the last projects that were loaded area all "FLO-2D Plugin" instead of the name of the GeoPackage or the Name of the Project.

![image](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/20424460/43a66eea-84ca-48c4-ad7b-43039e1cfe89)

**Comment by FLO-2DKaren** [comment]
Video https://flo-2d.sharefile.com/d-se153c50e6961422ea9dd4077c49ac364

**Comment by rpachaly** [comment]
I know the issue. This is going to be easy to fix.

**Comment by rpachaly** [comment]
> I know the issue. This is going to be easy to fix.

Not as easy as I imagined

**Comment by FLO-2DKaren** [comment]
Where is it even getting that name from?

**Comment by rpachaly** [comment]
Two points here: 

1. I had to set the Project Name as the same geopackage name to avoid potential bugs. So, I changed the settings dialog. Here is the new one:

![Image](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/39889306/944512af-35a1-4a83-b53f-1b64699b6d08)

2. I need to explicitly save the FLO-2D project on the recent projects when the FLO-2D save icon is clicked. What do you think about adding a FLO-2D image on FLO-2D projects?

![Image](https://github.com/FLO-2DSoftware/qgis-flo-2d-plugin/assets/39889306/c4ca6716-ad94-4386-b496-5c59b5ed1ab4)

**Comment by rpachaly** [comment]
> Where is it even getting that name from?

Currently, we cannot get the name of the .qgz inside the geopackage. So I hardcoded the "FLO-2D-Project" name into the plugin, and it turned out to be a bad idea. 

Setting a custom name for the geopackage and project makes it hard to track the .qgz inside the geopackage. So, I set the same geopackage name as the project inside the geopackage. It makes it easier to get the .qgz name. 

Is my explanation clear?
